transport/layer.py

```diff
@@ -48,13 +52,13 @@ def register_light_client(self, raiden_api: 'RaidenAPI', registration_data: dict
         """
 
     @abstractmethod
-    def add_light_client(self, light_client_transport: TransportNode):
+    def construct_full_node(self, config):
         """
-        Add a light client transport node to the layer.
+         This function must return a subtype of TransportNode
         """
 
     @abstractmethod
-    def remove_light_client(self, light_client_transport: TransportNode):
+    def construct_light_clients_nodes(self, config):
         """
-        Remove a light client transport node from the layer.
+         This function must return a list of objects that are subtype of TransportNode
```

Comment:
we should probably mention the fact that they should correspond to light client transport nodes, rather than just any transport node (even if the class was the same)

Reply:
done